.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: cpp
+compoiler: gcc
+dist: bionic
+
+before_install:
+  - sudo apt-get update
```

Comment:
You are missing upgrade also. Update only refreshes manifests.

Reply:
Added upgrade.